<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndirectCalibration</class>
 <widget class="QWidget" name="IndirectCalibration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_10">
   <item>
    <widget class="QGroupBox" name="gbInput">
     <property name="title">
      <string>Input</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <item>
       <layout class="QHBoxLayout" name="loCalibInputFile">
        <item>
         <widget class="MantidQt::MantidWidgets::MWRunFiles" name="leRunNo" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>41</verstretch>
           </sizepolicy>
          </property>
          <property name="label" stdset="0">
           <string>Run No</string>
          </property>
          <property name="multipleFiles" stdset="0">
           <bool>true</bool>
          </property>
          <property name="algorithmAndProperty" stdset="0">
           <string/>
          </property>
          <property name="fileExtensions" stdset="0">
           <stringlist>
            <string>.raw</string>
           </stringlist>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbPlotRaw">
          <property name="toolTip">
           <string>Plot first detector spectra</string>
          </property>
          <property name="text">
           <string>Plot Raw</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="loCalibScaleFactor">
        <item>
         <widget class="QCheckBox" name="ckScale">
          <property name="text">
           <string>Scale by factor:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spScale">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <double>999.990000000000009</double>
          </property>
          <property name="value">
           <double>1.0</double>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbCalibration">
     <property name="title">
      <string>Calibration</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_20">
      <item>
       <layout class="QVBoxLayout" name="propertiesCalibration"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="plotCalibration"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbResolution">
     <property name="title">
      <string>Resolution</string>
     </property>
     <layout class="QHBoxLayout" name="cal_plots">
      <item>
       <layout class="QVBoxLayout" name="loResolutionOptions">
        <item>
         <layout class="QGridLayout" name="loResolutionOptions2">
          <item row="2" column="0">
           <widget class="QCheckBox" name="ckResolutionScale">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Scale RES:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="ckCreateResolution">
            <property name="toolTip">
             <string>Create RES file</string>
            </property>
            <property name="text">
             <string>Create RES File</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QCheckBox" name="ckSmoothResolution">
            <property name="toolTip">
             <string>Apply WienerSmooth algorithm to resolution</string>
            </property>
            <property name="text">
             <string>Smooth RES</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="spResolutionScale">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximum">
             <double>999.990000000000009</double>
            </property>
            <property name="value">
             <double>1.0</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="plotResolution"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbOutput">
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="ckPlot">
        <property name="toolTip">
         <string>Plot resulting calibration file when created.</string>
        </property>
        <property name="text">
         <string>Plot Result</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_0">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="ckSave">
        <property name="text">
         <string>Save Result</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::MWRunFiles</class>
   <extends>QWidget</extends>
   <header>MantidQtMantidWidgets/MWRunFiles.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>ckResolutionScale</sender>
   <signal>toggled(bool)</signal>
   <receiver>spResolutionScale</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ckScale</sender>
   <signal>toggled(bool)</signal>
   <receiver>spScale</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
